### PR TITLE
docs: add docs for MongoDB timeseries outputstream

### PR DESCRIPTION
**Pull request by tim-golioth**
The branch for this PR was created from the docs/325 branch (in #267). Once #267 is merged, we can rebase main onto this branch.

### CODE REVIEW

File: docs/cloud/2-output-streams/1-mongo/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+id: overview
+title: MongoDB Overview
+slug: /cloud/output-streams/mongo
```
Comment (dylanswartz) [inline docs/cloud/2-output-streams/1-mongo/README.md:4]:
```suggestion
slug: /cloud/output-streams/mongodb-timeseries
```

File: docs/cloud/2-output-streams/1-mongo/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  to send your events
+* **MongoDB measurement name**: The name of the MongoDB measurement
+
+![Complete create Output Stream form](./assets/create-outputstream-form.png)
+
+## Example Data
+
+Golioth Output Streams use the [Cloud Events](https://cloudevents.io) format.
+Because MongoDB is a timeseries database, we transform events from the
+CloudEvents format to the MongoDB [line protocol](https://docs.influxdata.com/influxdb/cloud-serverless/reference/syntax/line-protocol/)
```
Comment (dylanswartz) [inline docs/cloud/2-output-streams/1-mongo/README.md:80]: @tim-golioth This is left over from InfluxDB line protocol transformation. Is there an alternative description to transforming into a MongoDB Time Series Document or can we delete this section?

File: docs/cloud/2-output-streams/1-mongo/_category_.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+label: 'MongoDB'
```
Comment (dylanswartz) [inline docs/cloud/2-output-streams/1-mongo/_category_.yml:1]:
```suggestion
label: 'MongoDB Time Series'
```

File: docs/cloud/2-output-streams/1-mongo/README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+* **Name**: The name you want to give to your Output Stream
+* **Event Type Filter**: select `DEVICE_STREAM_TYPE` from the list of types (the
+  MongoDB Output Stream is currently limited to stream events only)
+* **MongoDB URL**: Paste the `Cluster URL` you copied from your MongoDB Cloud org
+  settings
+* **MongoDB token**: Paste the MongoDB Cloud API token
+* **MongoDB bucket**: the name of an existing MongoDB Cloud bucket where you want
+  to send your events
+* **MongoDB measurement name**: The name of the MongoDB measurement
```
Comment (dylanswartz) [inline docs/cloud/2-output-streams/1-mongo/README.md:72]:
```suggestion
* **Name**: The name you want to give to your Output Stream
* **Event Type Filter**: select `DEVICE_STREAM_TYPE` from the list of types (the
  MongoDB Output Stream is currently limited to stream events only)
* **Connection String**: Paste the `Connection String` you copied for your MongoDB cluster
* **Database**: the name of an existing MongoDB database where you want
  to send your events
* **Collection**: The time series collection to store your events
* **Timefield**: the name of the field containing the date in your time series document
* **Metafield**: the name of the field containing metadata in your timeseries document (your deviceId and projectId will be populated here) 
```

**Review by dylanswartz — CHANGES_REQUESTED** [review]

**Comment by szczys** [comment]
@tim-golioth I think there is an image that didn't make it into a commit:

```
Error: Image docs/cloud/2-output-streams/1-mongo/assets/select-outputstream.png used in docs/cloud/2-output-streams/1-mongo/README.md not found.
    at async Promise.all (index 0)
[ERROR] Client bundle compiled with errors therefore further build is impossible.
```

**Comment by tim-golioth** [comment]
> @tim-golioth I think there is an image that didn't make it into a commit:
> 
> ```
> Error: Image docs/cloud/2-output-streams/1-mongo/assets/select-outputstream.png used in docs/cloud/2-output-streams/1-mongo/README.md not found.
>     at async Promise.all (index 0)
> [ERROR] Client bundle compiled with errors therefore further build is impossible.
> ```

yep, i think Dylan removed the link from the doc.

File: docs/cloud/2-output-streams/1-mongo/README.md (hunk 2 above)
Comment (tim-golioth) [reply docs/cloud/2-output-streams/1-mongo/README.md:80]: let me check, nice catch!

Comment (tim-golioth) [reply docs/cloud/2-output-streams/1-mongo/README.md:80]: @dylanswartz see update

File: docs/cloud/2-output-streams/1-mongo/README.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+| Metafield| string | ✅        | Name to use for the metafield in the BSON document (e.g. "metadata")|
+
+## Overview
+
+To use this integration, you need to:
+
+- Have a MongoDB instance (either a standalone instance or in Atlas, Mongo's managed service)
+- Create a MongoDB user that has a username and password
+- Create a timeseries collection in your MongoDB database
+- (Optional) Whitelist Golioth's IP address in your Atlas deployment
```
[the remark this answers is not in the document]
Comment (tim-golioth) [reply docs/cloud/2-output-streams/1-mongo/README.md:35]: yes, per slack discussion, the IP address to use is this:
`34.136.157.127/32`

File: docs/cloud/2-output-streams/1-mongo/README.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+* **Timefield**: the name of the field containing the date in your time series document
+* **Metafield**: the name of the field containing metadata in your timeseries document (your deviceId and projectId will be populated here)
+
+![Complete create Output Stream form](./assets/create-outputstream-form.png)
+
+## Example Data
+
+Golioth Output Streams use the [Cloud Events](https://cloudevents.io) format.
+Because MongoDB is a timeseries database, we transform events from the
+CloudEvents format to the MongoDB [line protocol](https://docs.influxdata.com/influxdb/cloud-serverless/reference/syntax/line-protocol/)
```
Comment (dylanswartz) [inline docs/cloud/2-output-streams/1-mongo/README.md:141]:
```suggestion
Golioth Output Streams utilize the [Cloud Events](https://cloudevents.io/) format. Therefore, it is necessary to transform events from the CloudEvents format into a MongoDB BSON (Binary JSON) document.
```

**Review by dylanswartz — CHANGES_REQUESTED** [review]

**Review by szczys — APPROVED** [review]
LGTM!

**Review by dylanswartz — APPROVED** [review]

**Pull request merged**